pkg/agent/openflow/pipeline.go

```diff
@@ -2704,3 +2655,233 @@ func (sl conjunctiveActionsInOrder) Less(i, j int) bool {
 	}
 	return sl[i].nClause < sl[j].nClause
 }
+
+// defaultDropFlows generates the default flows with drop action for some tables.
+func (f *featurePodConnectivity) defaultDropFlows() []binding.Flow {
```

Comment:
How about this?
```go
// realizePipelines sets nextID and missing action for every flow table in every pipeline and realize it on OVS bridge.
func (c *client) realizePipelines() {
	tableMissDropTables := map[string]struct{}{
		PipelineRootClassifierTable.GetName(): {},
		ClassifierTable.GetName():             {},
		SpoofGuardTable.GetName():             {},
	}

	for _, pipeline := range c.pipelines {
		tables := pipeline.ListAllTables()
		for i := range tables {
			var nextID uint8
			var missAction binding.MissActionType
			// For the last table in a pipeline, set the miss action to TableMissActionDrop.
			if pipeline.IsLastTable(tables[i]) {
				nextID = binding.LastTableID
				missAction = binding.TableMissActionDrop
			} else {
				nextID = tables[i+1].GetID()
				// For a table (not the last one) in a pipeline, if the table name is in tableMissDropTables, set the miss
				// action to TableMissActionDrop; otherwise set the miss action to TableMissActionNext.
				if _, ok := tableMissDropTables[tables[i].GetName()]; ok {
					missAction = binding.TableMissActionDrop
				} else {
					missAction = binding.TableMissActionNext
				}
			}
			tables[i].SetNext(nextID)
			tables[i].SetMissAction(missAction)
			c.bridge.CreateTable(tables[i], nextID, missAction)
		}
	}
}
```
Related comment is also added.
```
...
//  - Reference the new table in a feature in file pkg/agent/openflow/framework.go. The table can be referenced by multiple
//    features if multiple features need to install flows in the table. Note that, if the newly added table is not
//    referenced by any feature or the features referencing the table are all inactivated, then the table will not
//    be realized in OVS; if at least one feature referencing the table is activated, then the table will be realized
//    at the desired position in OVS pipeline.
//  - By default, the miss action of the new table is to next table. If the miss action needs to drop packets, add the
//    name of the new table to the map tableMissDropTables in func (c *client) realizePipelines() in current file.
...